class-wc-calypso-bridge-shared.php

```diff
@@ -125,6 +125,18 @@ public function add_scripts() {
 			'version'                      => WC_CALYPSO_BRIDGE_CURRENT_VERSION,
 		);
 
+		/**
+		 * `wc_calypso_bridge_shared_params` filter.
+		 *
+		 * This filter is used to add additional shared params to the bridge.
+		 *
+		 * @since 2.3.4
```

Comment:
Since 2.3.4 is already released, let's put x.x.x as a placeholder (it's an unofficial process at the moment, but on release we'll usually do a global search for `x.x.x.` and bump it to the release version)
```suggestion
		 * @since x.x.x
```